.mockery.yaml

```
@@ -170,9 +170,9 @@ packages:
       ConnectionsManager:
       ORM:
       Service:
-  github.com/smartcontractkit/chainlink/v2/core/services/feeds/proto:
+  github.com/smartcontractkit/chainlink-protos/orchestrator/feedsmanager:
     config:
-      dir: "{{ .InterfaceDir }}/../mocks"
+      dir: "./core/services/feeds/mocks"
```

It may not have an effect, but for consistency within this file we seem to omit the leading `./`:
```suggestion
      dir: "core/services/feeds/mocks"
```

---

https://github.com/smartcontractkit/chainlink/pull/14693/commits/95a5435f71088f79279f3a8729973160b7753cf3